skyportal/handlers/api/followup_request.py

```diff
@@ -0,0 +1,129 @@
+import arrow
+from marshmallow.exceptions import ValidationError
+
+from baselayer.app.access import auth_or_token
+from ..base import BaseHandler
+from ...models import DBSession, Instrument, Source, FollowupRequest, Token
+
+
+class FollowupRequestHandler(BaseHandler):
+    @auth_or_token
+    def post(self):
+        """
+        ---
+        description: Submit follow-up request.
+        requestBody:
+          content:
+            application/json:
+              schema: FollowupRequest
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - Success
+                    - type: object
+                      properties:
+                        id:
+                          type: string
+                          description: New follow-up request ID
+        """
+        data = self.get_json()
+        _ = Source.get_if_owned_by(data["source_id"], self.current_user)
+        data["start_date"] = arrow.get(data["start_date"]).datetime
```

Comment:
The arguments should be documented.

Reply:
We specify the schema, as in our other handlers, so these fields will automatically show up in the API docs

Reply:
The input fields do not show up automatically for get requests, do they?

Reply:
this is a POST handler...

Reply:
Look at any other POST handler -- we always use the schema to create the API docs. Look above here in this file:
```
        requestBody:
          content:
            application/json:
              schema: FollowupRequest
```

Reply:
You're right; thanks.